drive/types.bal

```diff
@@ -28,19 +28,19 @@ public type Configuration record {
     http:ClientSecureSocket secureSocketConfig?;
 };
 
-# Drive Info Record Type
+# Represents drive information.
 #
-# + kind - Identifies what kind of resource this is. Value: the fixed string "drive#about".  
-# + user - The authenticated user.
+# + kind - Identifies what kind of resource this is. Value: the fixed string "drive#about"
+# + user - The authenticated user
 # + storageQuota - The user's storage quota limits and usage. All fields are measured in bytes
-# + importFormats - A map of source MIME type to possible targets for all supported imports.
-# + exportFormats - A map of source MIME type to possible targets for all supported exports.
-# + maxImportSizes - A map of maximum import sizes by MIME type, in bytes.
-# + maxUploadSize - The maximum upload size in bytes.
-# + appInstalled - Whether the user has installed the requesting app.
-# + folderColorPalette - The currently supported folder colors as RGB hex strings.
-# + driveThemes - A list of themes that are supported for shared drives.
-# + canCreateDrives - Whether the user can create shared drives.
+# + importFormats - A map of source MIME type to possible targets for all supported imports
+# + exportFormats - A map of source MIME type to possible targets for all supported exports
+# + maxImportSizes - A map of maximum import sizes by MIME type, in bytes
+# + maxUploadSize - The maximum upload size in bytes
+# + appInstalled - Whether the user has installed the requesting app
```

Comment:
Is the "requesting app" an application that is requesting access to Google Drive, or is it an application that needs to be installed to run the API?

If it is the former, suggesting "Indicates whether the application that is requesting access is installed"

If it is the latter, suggesting "Indicates whether the requested application is installed."

Reply:
Changed to following 
```Indicates whether the requested application is installed.```
/Thanks